source/common/cardEeprom.c

```diff
@@ -150,8 +163,22 @@ uint32 cardEepromGetSize() {
 
 		if ( ((id >> 16) & 0xff) == 0xC2 ) { // Macronix
 			
-			if (device == 0x2211)
+			switch(device) {
+
+			case 0x2211:
 				return 128*1024;		//	1Mbit(128KByte) - MX25L1021E
+				break;
+			case 0x2017:
+				return 8*1024*1024;		//	64Mbit(8 meg)
+				break;
+			}
+		}
+
+		if (id == 0xffffff) {
+			int sr = cardEepromCommand(SPI_EEPROM_RDSR);
+			if (sr == 2) { // Pokémon Mystery Dungeon - Explorers of Sky
```

Comment:
Personal pet peeve, but naming games in the official repository feels wrong.

Reply:
Yes, I think the comment should be removed.

Reply:
Removed